driver/makedepend.ml

```diff
@@ -634,6 +635,11 @@ let main () =
          "<file> Read additional NUL separated command line arguments from \n\
          \      <file>"
   ];
+
+  if !Clflags.exclude_dir == false then begin
+    add_to_list first_include_dirs Filename.current_dir_name;
+  end;
+
```

Comment:
Physical equality `==` should only be used with good reason. Here, you wanted to use `=`. Similarly ` ... = trrue` or  `… = false` is a code smell. Also there is not the right location.